docs/release-notes/release-notes-0.9.0.md

```diff
@@ -0,0 +1,144 @@
+
+For full details of the scheme see: [Design for improving JoinMarket's resistance to sybil attacks using fidelity bonds](https://gist.github.com/chris-belcher/18ea0e6acdb885a2bfbdee43dcd6b5af/)
+
+This release implements all the features needed to add fidelity bonds to JoinMarket. Takers (via scripts such as `sendpayment.py` or `tumbler.py` or the Joinmarket-Qt app) will automatically give preference to makers who advertise fidelity bonds. Makers can optionally update their wallets to fidelity bond wallets. When a fidelity bond wallet is used with a yield generator script, it will automatically announce its fidelity bond publicly. Makers who don't create fidelity bonds by locking up bitcoins will still be chosen for coinjoins occasionally, but probably much less often than before.
+
+For full user documentation see the file [fidelity-bonds.md](../fidelity-bonds.md) in the repository.
+
+With realistic assumptions we have calculated that an adversary would need to lock up around 50000 bitcoins for 6 months in order to sybil attack the JoinMarket system with 95% success rate. Now that fidelity bonds are being added to JoinMarket for real we can see how the system behaves in practice.
+
+Fidelity bond coins cannot be yet be held in cold storage, but this is easy to add later because the JoinMarket protocol is set up in a way that the change would be backward-compatible.
+
+`199b571` Consider fidelity bonds when choosing makers
+
+`97b8b3b` Show fidelity bonds on orderbook watch html page
+
+`7a50c76` Announce yieldgenerator's fidelity bond
+
+`6b6fc4a` Handle fidelity bonds in client-server protocol
+
+`662f097` Write and update fidelity bond docs
+
+`eb0a738` Add interest rate option to config file
+
+`d4b3f70` Enable creation of fidelity bond wallets on cli
+
+`b9eab6e` Increase max locktime of fidelity bond wallets
+
+`a3b3cd4` Make fidelity bond wallets be native segwit
+
+`a3b775b` Increase default fee by 4x
+
+`9a372fd` Add getblockhash RPC method
+
+`e6c0847` Add calculate fidelity bond value function + tests
+
+`c70b12f` For timelock addrs use new pubkey foreach locktime
+
+`3dc8d86` Fix importprivkey on fidelity bond wallets
+
+`bbd3d1b` Print privacy warning when showing timelocked addr
+
+`1ea62a7` Fix bug with timelocked addrs in receive payjoin
+
+`4868343` Fix showutxos wallettool method for fidelity bonds
+
+`b27659c` Support fidelity bond wallet in Qt
+
+#### Minor Qt changes:
+
+Recent testing indicated that the Qt view update can
+be slow and profiling the code showed that this was due to
+re-calculation of wallet view hitting the backend python-bitcointx
+deserialization routines too hard. The obvious solution is to not
+recalculate the wallet view in a polling loop, but only when a change
+occurs in the wallet (though this can be optimized further, it's
+already much more responsive than before):
```

Comment:
Maybe could be also mentioned that now wallet updates (incoming transactions) will appear in Qt GUI almost immediately, not after up to 5 sec. delay.

Reply:
Oh yes, true. I guess I considered it a minor detail because there is, in many use cases, a few seconds delay anyway (bitcoin network propagation).

Reply:
In practice, from my tests, difference is huge. Also another minor detail (not worth mentioning I think) - wallet refresh closed already opened context menu, that was sometimes annoying if happened just before you tried to select "Copy address to clipboard".

Reply:
Oh for sure, I do see a big difference too, and I also saw that context menu thing. But "GUI is blocked" is separate from "there's a delay between new utxo arrives, Qt display gets updated with new utxo".